pages/reader/epub.vue

```diff
@@ -675,6 +492,52 @@ async function loadEPub() {
     percentage.value = book.locations.percentageFromCfi(location.start.cfi)
     currentCfi.value = location.start.cfi
   })
+
+  const allSegments = await collectAllSegments(book)
+  setTTSSegments(allSegments)
+}
+
+async function collectAllSegments(book: ePub.Book) {
+  const allSegments: { text: string, id: string, index: number }[] = []
```

Comment:
should move TTSSegment to `/types/tts.d.ts`
```suggestion
  const ttsSegments: TTSSegment[] = []
```